### PR TITLE
Coverings

### PR DESCRIPTION
**Pull request by gazuntype**
fill deh face!

### CODE REVIEW

File: Editor/Simulator/Simulator.cs [hunk 1]
```diff
@@ -687,7 +707,8 @@ public class Simulator : SimulatorBase {
         mesh.Clear();
         if (vertices.Count > 0 && indices.Count > 0) {
             mesh.SetVertices(vertices);
-            mesh.SetIndices(indices, MeshTopology.Triangles, 0, false);
+            GetIndicesWithCoverings(indices);
```
Comment (chornesays) [inline Editor/Simulator/Simulator.cs:710]: we should only do this if we need the coverings

**Review by chornesays — APPROVED** [review]
Dope!

My assumption is that since we're using pre-existing verts the UVs of the new mesh are being interpolated

**Comment by gazuntype** [comment]
correct assumption.

File: Editor/Simulator/Simulator.cs (hunk 1 above)
Comment (gazuntype) [reply Editor/Simulator/Simulator.cs:710]: it's only ever done once regardless.

**Pull request merged**